Added todo steps for when sphinx-autobuild is not found in the README (#532)
mollie/api-documentation: README.md

## Working at Mollie

Mollie is always looking for new talent to join our teams. We’re looking for inquisitive minds with good ideas and
strong opinions, and, most importantly, who know how to ship great products. Want to join the future of payments?
[Check out our vacancies](https://jobs.mollie.com).

## License

The documentation is licensed under the [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/?) license.

# FAQ

- [sphinx-autobuild is not found](#sphinx-autobuild-not-found)

## sphinx-autobuild not found

It might be that the location of the installed pip packages is not included in your `$PATH`.

> It's also recommended to use `python3` and `pip3` instead of the pre-installed python 2.7 version which comes with MacOS.

Reviewer: ```suggestion
It's also recommended to use `python3` and `pip3` instead of the pre-installed Python 2.7 version which comes with macOS.
```

The following steps install `python3` and `pip3` and add the package location to your `$PATH` variable

> 1. make sure you have homebrew installed (http://brew.sh)

Reviewer: Can you capitalize the first words of every sentence?

2. install python3, which comes with pip3: `brew install python3`
3. see if the executable works for both python3 and pip3: `which python3 && which pip3`. This command should print two paths
4. `pip3` installs its package executables on `/Users/<your-username>/Library/Python/<version>/bin`. Add this path to your `$PATH` variable. Add `export PATH=/Users/<your-username>/Library/Python/<version>/bin:$PATH"` to your `.zshrc` or `.bash_profile`.

> 5. You can choose to alias pip3 to pip. Add this to your `.zshrc` or `.bash_profile`: `alias pip=pip3`

Reviewer: ```suggestion
5. It might be convenient to alias `pip3` to `pip`. You can do so by adding the following line to your `.zshrc` or `.bash_profile`: `alias pip=pip3`
```

> 6. If you did step 5, `make install` will now work. If you didn't. Copy the make install command and replace `pip` with `pip3`.

Reviewer: ```suggestion
6. If you did step 5, `make install` should now work. If you didn't: copy the `make install` command and replace `pip` with `pip3`.
```

> 7. `make start` is now able to run smoothly.

Reviewer: ```suggestion
7. The command `make start` should now run smooth again.
```
